### ISSUE TITLE
Question about reproduce 

### ISSUE THREAD

**Issue by liujunyan691** [CAMMA-public/rendezvous#24]
Thank you for your work and sharing. I have read your papers on dataset splits and RDV, but I have found that the default training method in the code is different from the training method described in the paper. I have trained using both methods and have not been able to reproduce the results on cholect50 using RDV split  (29.9 on pytorch). Could you please provide the model weights corresponding to the 29.9 result on pytorch and describe the corresponding training strategy? (I saw in the previous answer that one of the reasons is the use of different augmentation style per epoch, which is different from the simplified dataloader in the git repo. But this description is too simplistic to reproduce it as a baseline for me). Can you please share your hyperparameters and describe which training strategy corresponds to 29.9？

**Comment by liujunyan691** [comment]
Hoping for your answer! Thanks again

**Comment by nwoyecid** [comment]
Please, use the provided weights. They give the same results as in the paper.

**Comment by ntlm1686** [comment]
> Please, use the provided weights. They give the same results as in the paper.

Could you please provide an evaluation script that supports the new data format, where the labels are only in the `labels` folder as JSON files?

**Comment by nwoyecid** [comment]
Ivtmetrics supports the new data format, check documentation for more
details

**Comment by labdeeman7** [comment]
I gave what worked for me here. https://github.com/CAMMA-public/rendezvous/issues/26#issuecomment-2796104646
